package.json

```diff
@@ -121,11 +121,21 @@
     "prepare": "husky install",
     "sourcemaps:api": "turbo run sourcemaps --filter=api"
   },
+  "lint-staged": {
+    "*.{js,ts,tsx}": [
```

Comment:
This is true. I have added more extensions that I found your project uses. Let me know if you would like any further changes. I could not add .json files for the linting as you do not have any configuration for it. You use various eslint-plugins but not eslint-plugin-json. Would you like me to add it? otherwise I can only run the formatter which is why I added a seperate script for it in the lint-stage